packages/react-vapor/src/components/actions/ActionBar.tsx

```diff
@@ -130,42 +117,69 @@ export class ActionBar extends React.Component<IActionBarProps, any> {
             );
         }
 
-        const actions =
-            primaryActions.length || secondaryActionsView || this.props.prompt ? (
-                <div className="coveo-table-actions">
-                    {primaryActions}
-                    {secondaryActionsView}
-                    {this.props.prompt}
-                </div>
-            ) : null;
+        const actions = this.props.prompt ? (
+            <this.Prompt />
+        ) : (
+            <>
+                {primaryActions}
+                {secondaryActionsView}
+            </>
+        );
 
-        const defaultContainerClasses = !this.props.removeDefaultContainerClasses
-            ? DEFAULT_ACTIONS_CONTAINER_CLASSES
-            : [];
+        return primaryActions.length || secondaryActionsView || this.props.prompt ? (
+            <div className="coveo-table-actions">{actions}</div>
+        ) : null;
+    };
 
-        const containerClasses = classNames(defaultContainerClasses, this.props.extraContainerClasses, {
-            'mod-deactivate-pointer': !!this.props.isLoading,
-            'small-actions-container': this.props.withSmallActions,
-        });
+    private Prompt: React.FunctionComponent = () =>
+        this.props.prompt?.options ? (
+            <div className="prompt">
+                <InlinePromptConnected {...this.props.prompt} />
+            </div>
+        ) : null;
 
-        const prefixContentElement: JSX.Element = this.props.prefixContent ? (
-            <Content {...this.props.prefixContent} />
+    private Filter: React.FunctionComponent = () =>
+        this.props.itemFilter ? (
+            <ItemFilter
+                label={this.props.itemFilterLabel}
+                item={this.props.itemFilter}
+                itemTooltipProps={this.props.itemTooltipProps}
+                onClear={this.props.clearItemFilter}
+                crop={this.props.itemFilterCropLength}
+            />
         ) : null;
+}
 
-        const divProps: React.HTMLProps<HTMLDivElement> = {
-            className: containerClasses,
-        };
-        if (this.props.width) {
-            divProps.style = {width: this.props.width};
-        }
+const mapStateToProps = (state: IReactVaporState, ownProps: IActionBarProps) => {
+    const actionBar: IActionBarState = _.findWhere(state.actionBars, {id: ownProps.id});
+    const prompt: IPromptState = _.find(state.prompts, ({id}) => id.indexOf(ownProps.id) >= 0);

```

Comment:
Bug fix is here.
```tsx
// before
const prompt: IPromptState = _.findWhere(state.prompts, {id: ownProps.id});
```
With `_.find` and `id.indexOf`, it will pickup `action-bar-connected` and `action-bar-connected_actionsDropdown` instead of just `action-bar-connected`.